### PR TITLE
Add links to YouTube channels of conferences as well

### PR DESCRIPTION
**Pull request by amadio**
What it looks like now.

screenshot

### CODE REVIEW

**Comment by amadio** [comment]
Converted to draft. Apparently `youtube.com/c/<channel>` doesn't always work...

File: talk/setup.tex [hunk 1]
```diff
@@ -234,6 +235,16 @@
   \href{#1}{\color{blue!50!white} cppreference}%
 }
 
+% Use as \youtubechannel{username}
+\newcommand\youtubechannel[1]{%
```
Comment (bernhardmgruber) [inline talk/setup.tex:239]:
```suggestion
% Use as \youtubechannel{channelname}
\newcommand\youtubechannel[1]{%
```

**Review by bernhardmgruber — APPROVED** [review]

**Comment by bernhardmgruber** [comment]
I also clicked all links and they work!

File: talk/setup.tex (hunk 1 above)
Comment (amadio) [reply talk/setup.tex:239]: Oops, fixing.

**Review by bernhardmgruber — APPROVED** [review]

**Pull request merged**